### ISSUE TITLE
Cannot open any files

### ISSUE THREAD

**Issue by oleblancpellerin** [kaitai-io/kaitai_struct_gui#10]
I cannot open any file.
Here is the console output:
```
Exception in thread "main" java.util.NoSuchElementException: None.get
        at scala.None$.get(Option.scala:349)
        at scala.None$.get(Option.scala:347)
        at io.kaitai.struct.format.KSVersion$.current(KSVersion.scala:65)
        at io.kaitai.struct.format.MetaSpec$.$anonfun$fromYaml$1(MetaSpec.scala:76)
        at io.kaitai.struct.format.MetaSpec$.$anonfun$fromYaml$1$adapted(MetaSpec.scala:74)
        at scala.Option.foreach(Option.scala:257)
        at io.kaitai.struct.format.MetaSpec$.fromYaml(MetaSpec.scala:74)
        at io.kaitai.struct.format.ClassSpec$.$anonfun$fromYaml$1(ClassSpec.scala:89)
        at scala.Option.map(Option.scala:146)
        at io.kaitai.struct.format.ClassSpec$.fromYaml(ClassSpec.scala:89)
        at io.kaitai.struct.format.ClassSpec$.fromYaml(ClassSpec.scala:224)
        at io.kaitai.struct.formats.JavaKSYParser$.fileNameToSpec(JavaKSYParser.scala:29)
        at io.kaitai.struct.formats.JavaKSYParser.fileNameToSpec(JavaKSYParser.scala)
        at io.kaitai.struct.visualizer.VisualizerPanel.compileKSY(VisualizerPanel.java:99)
        at io.kaitai.struct.visualizer.VisualizerPanel.parseFileWithKSY(VisualizerPanel.java:126)
        at io.kaitai.struct.visualizer.VisualizerPanel.loadAll(VisualizerPanel.java:75)
        at io.kaitai.struct.visualizer.MainWindow.main(MainWindow.java:26)
```

**Comment by GreyCat** [comment]
@djeezuss Can you clarify what ksc are you using? I can't reproduce it, and it looks like it's the problem with kaitai-struct-compiler having no KSVersion object initialized for some reason.

**Comment by djeezuss** [comment]
I am using KSC 0.8

**Comment by GreyCat** [comment]
Can you tell me more about your environment — OS, JVM, classpath? Without a repro, unfortunately, the only thing I can think of is getting there with a debugger and checking what's going on with that variable and why is it empty.

**Comment by djeezuss** [comment]
OS: Windows 10 64bit
```
java version "1.8.0_241"
Java(TM) SE Runtime Environment (build 1.8.0_241-b07)
Java HotSpot(TM) Client VM (build 25.241-b07, mixed mode)
```
As for the classpath, I do not know how to check that?

**Comment by generalmimon** [comment]
I can reproduce the exact same issue (using Git Bash on Windows 10 x64, Java 1.8.0_241), a quick insight shows that the error in https://github.com/kaitai-io/kaitai_struct_gui/issues/10#issue-555872323 is connected with using `ks-version` key in the KSY (I was lucky that I chose such a spec for testing).

But I really can't open _any other KSY format_ (for example, I tried [utf8_string](http://formats.kaitai.io/utf8_string/index.html) which doesn't use any fresh features). It prints this:

```
Exception in thread "main" java.lang.NullPointerException
        at org.mdkt.compiler.InMemoryJavaCompiler.compileAll(InMemoryJavaCompiler.java:80)
        at org.mdkt.compiler.InMemoryJavaCompiler.compile(InMemoryJavaCompiler.java:126)
        at io.kaitai.struct.visualizer.VisualizerPanel.parseFileWithKSY(VisualizerPanel.java:138)
        at io.kaitai.struct.visualizer.VisualizerPanel.loadAll(VisualizerPanel.java:75)
        at io.kaitai.struct.visualizer.MainWindow.main(MainWindow.java:24)
```

Seems to have broken with merging https://github.com/kaitai-io/kaitai_struct_gui/pull/9.

BTW, some usage instructions in README would come in handy. I guessed I have to clone the repo first (not obvious, people might think that the package get installed automatically with `mvn install`), then launched the build commands described in README (that worked fine). But then I didn't know how to run the program. So I opened the promising `target` folder and see a .jar file. I launched it with a double-click, but just an empty hex dump showed up, with no file inputs which I would expect from a GUI. So I guessed that this wasn't the way. I searched on Google how to run a .jar package and found `java -jar [filename].jar`. OK, so let's ask the jar for some `--help`, right? I see, this doesn't work either. In the end, I had to look into the source code where I finally found the answer.
